### PR TITLE
(fix) Change expectation for jwt params

### PR DESCRIPTION
**Pull request by dvc94ch**

### CODE REVIEW

File: project-service/src/models/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import Sequelize from 'sequelize';
 import configs from '../config.js';
 
-let env = process.env.NODE_ENV || 'development';
+let env = process.env.NODE_ENV || 'test';
```
Comment (wezleytsai) [inline project-service/src/models/index.js:6]: So what would the environment be if i do docker-compose? `production`? How do I run on `development` database?

File: event-system/src/subscribe.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ let router = express.Router();
 
 router.get('/subscribe/:namespace/:room', (req, res) => {
   let client = redis.createClient();
+  req.socket.setTimeout(Number.MAX_SAFE_INTEGER);
```
Comment (wezleytsai) [inline event-system/src/subscribe.js:9]: Could you explain what this is for?

Comment (dvc94ch) [inline event-system/src/subscribe.js:9]: so sending the keep-alive header tells the browser to keep the connection open. this line tells the server to keep the connection open. But it probably shouldn't be part of this PR...
